AndroidAnnotations/functional-test-1-5/src/test/java/org/androidannotations/test15/SeekBarChangeListenedActivityTest.java

```diff
@@ -38,13 +38,16 @@
 	public void setUp() {
 		activity = setupActivity(SeekBarChangeListenedActivity_.class);
 		seekBar = (SeekBar) activity.findViewById(R.id.seekBar1);
-		shadowSeekBar = shadowOf_(seekBar);
+		shadowSeekBar = (ShadowSeekBar) extract(seekBar);
```

Comment:
We need to get the reference to the shadow to access the `getOnSeekBarChangeListener()` method, which is not present in the normal class.